coralproject/talk
# [CORL-491] Support event alias loginPrompt

## What does this PR do?
- Support event alias loginPrompt
- Add instructions to handle login prompts in SSO

## How do I test this PR?

Try to add this to `embed/index.html`. Disable logins except SSO. This should redirect you whenever a login is required in the stream.

```js
var embed = Coral.createStreamEmbed({
  // Don't forget to include the parameters from the
  // "Embed On Your Site" section.
  events: function(events) {
    events.on("loginPrompt", function() {
      // Redirect user to a login page.
      location.href = "http://example.com/login";
    });
  },
});
```
